kanidmd/src/lib/constants/schema.rs

```diff
@@ -175,7 +175,7 @@ pub const JSON_SCHEMA_ATTR_RADIUS_SECRET: &str = r#"{
         "radius_secret"
       ],
       "syntax": [
-        "RADIUS_UTF8STRING"
+        "SECRET_UTF8STRING"
```

Comment:
Because previously it was "radius" now it's "secret". The idea was originally that radius string (now secret string) holds a string that can be set into the server, but NEVER yielded in a search response, and can only be read via specific apis to prevent accidental disclosure. It turned out that I needed the same behaviour as radius string in oauth2 so it made sense to rename this rather than adding a duplicate field with the same behaviours.